c7n/resources/dynamodb.py

```diff
@@ -192,8 +170,8 @@ class SetStream(BaseAction, StatusFilter):
     permissions = ("dynamodb:UpdateTable",)
 
     def process(self, tables):
-        tables = self.filter_table_state(
-            tables, self.valid_status)
+        tables = self.filter_resources(
+            tables, 'TableStatus', self.valid_status)
         if not len(tables):
             self.log.warning("Table not in ACTIVE state.")
```

Comment:
can ditch the warning here, the filter_resources automatically log/warns if there are any resources filtered.

Reply:
Nice! Updated in 458af89.